lib/preboots/setup.js

```diff
@@ -2,29 +2,38 @@
 
 var wrhs = require('warehouse-models');
 var Feedsme = require('../feedsme');
-var Datastar = require('datastar');
-
+const { DynamoDB } = require('aws-sdk');
+var dynamo = require('dynamodb-x');
+const AwsLiveness = require('aws-liveness');
+const liveness = new AwsLiveness();
 var HttpsAgent = require('https').Agent;
 var HttpAgent = require('http').Agent;
 
-module.exports = function setup(app, options, next) {
+module.exports = function setup(app, options = { dynamo: {} }, next) {
   const ensure = app.config.get('ensure') || options.ensure;
   //
   // Setup the feedsme instance and other helpers.
   //
-  app.datastar = options.datastar || new Datastar(app.config.get('datastar') || {
-    config: app.config.get('cassandra')
-  });
-  app.models = options.models || wrhs(app.datastar);
+  const region = app.config.get('AWS_REGION') || app.config.get('dynamo:region') || options.dynamo.region;
+  // Used mainly for localstack usage
+  const endpoint = app.config.get('DYNAMO_ENDPOINT') || app.config.get('dynamo:endpoint') || options.dynamo.endpoint;
+  const driver = new DynamoDB({ region, endpoint });
+
+  dynamo.dynamoDriver(driver);
+  app.dynamo = dynamo;
+  app.models = options.models || wrhs(app.dynamo);
```

Comment:
Is there a case where models might already be passed in? it'd be detached from app.dynamo at that point right?

Reply:
not sure, if not, then this should just be set to `wrhs(app.dynamo)` right?